src/dataflow-types/src/types.rs

```diff
@@ -775,6 +776,16 @@ impl fmt::Display for MzOffset {
     }
 }
 
+impl Add<i64> for MzOffset {
+    type Output = MzOffset;
+
+    fn add(self, x: i64) -> MzOffset {
+        MzOffset {
+            offset: self.offset + x,
```

Comment:
What is this commit here for? It doesn't appear to be fixing anything. Does this need to check for overflow? I have not heard of the MzOffset type until now so I'm not quite qualified to review this commit.

Reply:
it does not. this is just a commit that slots in the Add implementation before it gets used extensively by later commits that do exactly once sinks stuff